qiskit/providers/aer/aer_snapshots.py

```diff
@@ -0,0 +1,273 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017.
```

Comment:
The copyright should be `(C) Copyright IBM 2018, 2019.`

Reply:
Fixed!